lib/Maskings/XifyFront.cpp

```diff
@@ -0,0 +1,78 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
The whole `XifyFront` approach will not work when there are unique indexes on the data.
Because the `XifyFront` may create the same values, so reimporting the xified data will fail with "unique constraint violated".

Reply:
Yes, we need some additional checksum if one uses an index.